zerver/migrations/0386_fix_attachment_caches.py

```diff
@@ -60,5 +60,14 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
+        migrations.AlterField(
+            model_name="archivedattachment",
+            name="messages",
+            field=models.ManyToManyField(
+                related_name="attachment_set",
+                related_query_name="attachment",
+                to="zerver.ArchivedMessage",
+            ),
+        ),
```

Comment:
This could be rebased all the way back to where the field was added.

```diff
--- a/zerver/migrations/0067_archived_models.py
+++ b/zerver/migrations/0067_archived_models.py
@@ -141,7 +141,11 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name="archivedattachment",
             name="messages",
-            field=models.ManyToManyField(to="zerver.ArchivedMessage"),
+            field=models.ManyToManyField(
+                related_name="attachment_set",
+                related_query_name="attachment",
+                to="zerver.ArchivedMessage",
+            ),
         ),
         migrations.AddField(
             model_name="archivedattachment",
```